src/methods/call.js

```diff
@@ -432,6 +432,8 @@ export function OrdinaryCallEvaluateBody(
             invariant(c instanceof Value || c instanceof AbruptCompletion);
             return c;
           }
+        } catch (e) {
+          throw e;
```

Comment:
This doesn't mean anything, does it?
A rethrow catch clause is equivalent to no catch clause.

Reply:
Opps, that was left in from debugging, I'll remove.